adafruit_is31fl3731/__init__.py

```diff
@@ -262,7 +282,7 @@ def audio_play(self, sample_rate, audio_gain=0, agc_enable=False, agc_fast=False
         )
         self._mode(_AUDIOPLAY_MODE)
 
-    def blink(self, rate=None):
+    def blink(self, rate: int = None) -> int:
```

Comment:
rate can be `Optional[int]`

This function can return None so it's return type can also be `Optional[int]`